ReSwift/CoreTypes/MainStore.swift

```diff
@@ -38,14 +38,14 @@ public class MainStore<State: StateType>: Store {
         self.state = state
 
         // Wrap the dispatch function with all middlewares
-        self.dispatchFunction = middleware.reverse().reduce(self._defaultDispatch) {
-            dispatchFunction, middleware in
-                return middleware(self.dispatch, { self.state })(dispatchFunction)
+        self.dispatchFunction = middleware.reverse().reduce(_defaultDispatch) {
+            function, middleware in
+                return middleware(self.dispatch, { self.state })(function)
         }
```

Comment:
this was from a previous change I'd been experimenting with earlier, i'll change it back.